src/main/java/com/xceptance/neodymium/common/browser/BrowserData.java

```diff
@@ -32,16 +32,20 @@ public class BrowserData extends Data
 
     private List<RandomBrowsers> classRandomBrowsersAnnotation;
 
+    private Class<?> testClass;
+
     private static final String SYSTEM_PROPERTY_BROWSERDEFINITION = "browserdefinition";
 
     public BrowserData(Class<?> testClass)
     {
         this();
+        this.testClass = testClass;
```

Comment:
`this.testClass = testClass;` is also part of `initClassAnnotationsFor(Class<?> testClass)` in the next line.
Please remove one of them.

Reply:
done